content/cloud-docs/run/cli.mdx

```diff
@@ -191,11 +192,19 @@ Do you want to perform these actions in workspace "docs-workspace"?
 Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
 ```
 
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+### Non-Interactive Workflows
+
+External systems cannot run the traditional apply workflow because Terraform requires console input from the user to approve plans. We recommend using the [API-driven Run Workflow](/cloud-docs/run/api) for non-interactive workflows when possible.
 
-Remote applies use the configuration code from the local working directory, but use the variable values from the specified workspace.
+If you prefer to use the CLI in a non-interactive environment, we recommend first running a [speculative plan](/cloud-docs/run#speculative-plans) to preview the changes Terraform will make to your infrastructure. Then, use one of the following approaches with the `-auto-approve` flag based on the [execution mode](/cloud-docs/workspaces/settings#execution-mode) of your workspace. The [`-auto-approve`](/cli/commands/apply#auto-approve) flag skips prompting you to approve the plan.
+
+- **Local Execution:**  Save the approved speculative plan and then run `terraform apply -auto-approve` with the saved plan.
+- **Remote Execution:** Terraform Cloud does not support uploading saved plans for remote execution, so we recommend running `terraform apply -auto-approve` **immediately** after approving the speculative plan to prevent the plan from becoming stale. The [`-auto-approve`](/cli/commands/apply#auto-approve) flag skips prompting for approval.
```

Comment:
```suggestion
- **Remote Execution:** Terraform Cloud does not support uploading saved plans for remote execution, so we recommend running `terraform apply -auto-approve` **immediately** after approving the speculative plan to prevent the plan from becoming stale. 
```